website/content/en/highlights/2023-08-30.md

```diff
@@ -0,0 +1,33 @@
+---
```

Comment:
Just a note that the filename here should include a reasonable slug after the date. See the other files in `website/content/en/highlights/` for examples.